app/views/layouts/_flash_messages.html.erb

```diff
@@ -7,11 +7,11 @@
     </div>
   </div>
 <% end %>
-<% if flash[:error] %>
+<% if flash[:alert] %>
```

Comment:
Switched to `alert` because `redirect_to` only accepts `notice` and `alert` as built-in flash messages.